server/graphql/v2/object/Host.js

```diff
@@ -427,6 +442,113 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          accounts: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          from: {
+            type: GraphQLString,
+            description: "Inferior date limit in which we're calculating the contribution stats",
+          },
+          to: {
+            type: GraphQLString,
+            description: "Superior date limit in which we're calculating the contribution stats",
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the contribution stats.');
+          }
+          const where = { HostCollectiveId: host.id, kind: 'CONTRIBUTION' };
+          const dateRange = dateRangeHelper(args.from, args.to);
+          if (dateRange) {
+            where.createdAt = dateRange;
+          }
+          if (args.accounts) {
+            const collectiveIds = args.accounts.map(account => account.legacyId);
```

Comment:
What if someone makes a query like:

```gql
host(slug: "...") {
  contributionStats(accounts: [{ slug: "xxxx" }, { slug: "yyyy"}]) {
    ...
  }
}
```

Reply:
Reference inputs are meant to be processed with reference helpers.